dotCMS/src/main/java/com/dotcms/rest/api/v1/page/ResponseEntityPageView.java

```diff
@@ -0,0 +1,10 @@
+package com.dotcms.rest.api.v1.page;
```

Comment:
@gavriella-dotcms what is this class for? I see some postman tests broken. Might it be related?